website/pages/how-to-guides/attack-surface-management-with-graph.md

```diff
@@ -0,0 +1,237 @@
+---
+title: How to use CloudQuery for Attack Surface Management and Graph Visualization
+tag: tutorial
+description: >-
+  How to use CloudQuery for Attack Surface Management (ASM) and Graph Visualization with Neo4j
+author: jsonkao
+---
+
+import { HowToGuideHeader } from "../../components/HowToGuideHeader"
+
+<HowToGuideHeader/>
+
+In this guide, we will demonstrate how to set up CloudQuery for customizable Attack Surface Management (ASM) and how to get started with utilizing graph visualization to execute security queries.  With cloud infrastructure and security, graph visualization can be used effectively to understand relationships between assets, attack paths, and attack surface to aid with Attack Surface Management and improving security posture of your organization.
```

Comment:
I'd suggest placing one of the screenshots at the top of the post to help grab attention and give the reader an idea of what they'll get

Reply:
Yeah the graph screen shots are pretty cool!